dashboard/src/components/Config/PkgRepoList/PkgRepoForm.tsx

```diff
@@ -767,7 +769,8 @@ export function PkgRepoForm(props: IPkgRepoFormProps) {
                           checked={repo.namespaceScoped || isNamespaceScoped}
                           onChange={handleRepoScopeChange}
                           disabled={
-                            !!repo && isGlobalNamespace(namespace, plugin?.name, currentNsConfig)
+                            !!repo.name ||
+                            isGlobalNamespace(namespace, plugin?.name, currentNsConfig)
```

Comment:
This is the actual change, the condition was wrong 🤦 . We want to disable the field either if the current namespace is already global (so we don't have much to choose) OR if there is a repo being edited.

Reply:
Right. If the context namespace is already global, the user does not have a way to select a namespace. That is done through the context dialog.